cypress/e2e/0-JoinBet/footer/footer.cy.js

```diff
@@ -24,59 +52,46 @@ describe('Localization. Casino. Footer.', () => {
     })
 })
 
+beforeEach(()=>{
+    cy.authenticateUrl('https://st2-gdjsmns.hk/ru/casino/lobby')
+})
```

Comment:
@ElKalinin Дружище, нужно beforeEach положить внутрь describe:

```suggestion
  describe('[Казино] Авторизация', () => {
    beforeEach(() => {
        cy.authenticateUrl('https://st2-gdjsmns.hk/ru/casino/lobby')
```